apps/teamsjs-cc-app/templates/appPackage/aad.template.json

```diff
@@ -0,0 +1,109 @@
+{
+    "id": "{{state.fx-resource-aad-app-for-teams.objectId}}",
+    "appId": "{{state.fx-resource-aad-app-for-teams.clientId}}",
+    "name": "{{config.manifest.appName.short}}-aad",
+    "accessTokenAcceptedVersion": 2,
+    "signInAudience": "AzureADMyOrg",
+    "optionalClaims": {
+        "idToken": [],
+        "accessToken": [
+            {
+                "name": "idtyp",
+                "source": null,
+                "essential": false,
+                "additionalProperties": []
+            }
+        ],
+        "saml2Token": []
+    },
+    "requiredResourceAccess": [
+        {
+            "resourceAppId": "Microsoft Graph",
+            "resourceAccess": [
+                {
+                    "id": "User.Read",
```

Comment:
Is there a specific test account or test tenant that needs to be used with this application? I assume you didn't actually get graph tokens for @microsoft.com?

Reply:
(If so, instructions on how to use that may be a good thing to include in the README)

Reply:
Deleting the templates folder... I think it was generating when I provisioned same as the .backup